usaspending_api/download/v2/base_download_viewset.py

```diff
@@ -58,9 +58,14 @@ def post(
             )
             return self.get_download_response(file_name=filename)
 
-        json_request = request_type.value["validate_func"](request.data)
-        json_request["request_type"] = request_type.value["name"]
-        ordered_json_request = json.dumps(order_nested_object(json_request))
+        if validator_type is not None:
+            validator = validator_type(request.data)
+            json_request = validator.json_request
+        else:
+            json_request = request_type.value["validate_func"](request.data)
+            json_request["request_type"] = request_type.value["name"]
+        json_request = order_nested_object(json_request)
```

Comment:
What happens if neither a `validator_type` or `json_request` is passed in? Should `json_request` still be initialized?

Reply:
Every download should have at least one of those. They are not user provided, but provided when we define the endpoint.